docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,48 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known challenge in entity resolution is people being reported multiple
+times at different addresses. This can occur when family structures are
+complex and children spend time at multiple households. A related
+challenge occurs with college students, who are sometimes counted both at their
+university and at their guardian’s home address.
+
+To facilitate this type of error, we have simulants assigned to guardians
+within the simulation. Sometimes, those guardians may live at
+different addresses than their dependents. In this case, there is an
+opportunity for duplication. Since this mechanism occurs within the
+simulation, there is a natural maximum on how many simulants can be duplicated
+in this way.
+
+Guardian-based duplication is applied to two mutually exclusive categories of
+simulants based on age and GQ status: Simulants younger than 18 (<18) and not
+in GQ and simulants under 24 (<24) and in college GQ.
```

Comment:
```suggestion
simulants based on age and GQ status: Simulants younger than 18 and not
in GQ and simulants under 24 and in college GQ.
```
We needed to make sure to be extra clear and precise for the engineers, but I think writing out these clarifications will mostly just be distracting for users.